code/game/objects/items/toys.dm

```diff
@@ -396,6 +396,10 @@
 	item_state = "armblade"
 	attack_verb = list("pricked", "absorbed", "gored", "stung")
 	w_class = W_CLASS_MEDIUM
+	/obj/item/toy/foamblade/suicide_act(mob/user)
```

Comment:
What is this tabulation

Reply:
In addition to that the lack of a line break makes me angry

Reply:
did I do it wrong